app/controllers/application_controller.rb

```diff
@@ -11,6 +11,7 @@ class ApplicationController < ActionController::Base
   before_action :set_request_context
   after_action :set_body_class_header
   after_action :set_csp_header
+  after_action :updated_last_visited_on!
```

Comment:
I don't know if you want this to be a `before_action` 🤷

Reply:
This is fine.